database/db.go

```diff
@@ -59,16 +61,31 @@ func (d dbLogger) AfterQuery(q *pg.QueryEvent) {
 
 // New returns a new DB object which wraps
 // a connection to the database specified in config
-func New(config DBConfig) DB {
+func New(config DBConfig, logger *log.Logger) DB {
```

Comment:
prefer passing logger in as a param of config versus changing the signature of this method